src/main/java/ch/njol/skript/util/chat/ChatMessages.java

```diff
@@ -506,11 +506,19 @@ public static String stripStyles(String text) {
 		}
 		String plain = sb.toString();
 		
-		// To be extra safe, strip <, >, § and &; protects against bugs in parser
+		// To be extra safe, strip valid <, >, §, & and hex codes <#123456>; protects against bugs in parser
 		if (Utils.HEX_SUPPORTED) // Strip '§x'
 			plain = plain.replace("§x", "");
-		plain = plain.replace("<", "").replace(">", "").replace("§", "").replace("&", "");
+		/*
+			1st Replace: strips json
+			2nd Replace: strips colors using <color name> (Note: this regex expr repalce any two words surrounded by <> such as <any text>)
+			3rd Replace: strips colors & or §
+			4th Replace: strips colors using hex codes
+		*/
+		plain = plain.replaceAll("<(link|url|run command|suggest command|tooltip):(.*?)>", "$2").replaceAll("<\\w+? ?\\w+?>", "").replaceAll("(?i)[&§][0-9a-folkrnm]", "").replaceAll("(?i)<#[0-9a-z]{6}>", "");
```

Comment:
You don't need to worry about stripping HEX codes. They will have been parsed by this point and formatted like `§x§1§2§3§4§5§6` so you might want to make sure they are covered properly when you replace &/§ symbols. Ideally you should only do that when HEX codes are enabled and when it is a valid HEX tag (e.g. search for that kind of formatted HEX string).